.github/CODEOWNERS

```diff
@@ -1,6 +1,16 @@
 # Instructions for CODEOWNERS file format and automatic build failure notifications:
 # https://github.com/Azure/azure-sdk/blob/main/docs/policies/opensource.md#codeowners
 
+# For Client Libraries and Management Libraries, a service's entry MUST be inserted into their designated section alphabetically by service label and formatted as:
+
+####################
+# PRLabel: %<Service-Label>
+/sdk/<your-service-library-directory>/                            @fakeUser1 @Azure/fakeTeam1
```

Comment:
We cannot have entries here otherwise it will break validation. If we want to document things use the link at the top of the file here and provide examples in there.

Reply:
@weshaggard Would it be validation breaking to stub this in as a comment rather than a false entry? My concern is that since editors are already accustomed to updating the CODEOWNERs file prior to needing to meet formatting requirements, they won't re-review the documentation in the link.

Reply:
I don't think that the new requirement of alphabetical ordering is clearly implied from the other entries. It's also unclear if the indentation changes are consequential formatting changes.

Reply:
The ordering is also not enforced by anything, so it isn't a requirement currently it is just a recommendation to help with inserting new entries easier. Either way my thought is that we put any docs in the referenced link and as long as the entries are commented out in the codeowners file I don't mind it there, I guess.